daprdocs/content/en/contributing/contributing-overview.md

```diff
@@ -8,11 +8,23 @@ description: >
 ---
 
 Thank you for your interest in Dapr!
-This document provides the guidelines for how to contribute to the [Dapr project](https://github.com/dapr) through issues and pull-requests. Contributions can also come in additional ways such as engaging with the community in community calls, commenting on issues or pull requests and more.
+This document provides the guidelines for how to contribute to the [Dapr project](https://github.com/dapr) through issues and pull requests. Contributions can also come in additional ways such as engaging with the community in community calls, commenting on issues or pull requests, and more.
 
 See the [Dapr community repository](https://github.com/dapr/community) for more information on community engagement and community membership.
 
-> If you are looking to contribute to the Dapr docs, please also see the specific guidelines for [docs contributions]({{< ref contributing-docs >}}).
+## Dapr Repository Index
```

Comment:
Can you add dapr/cli to this list as well? Then we have all the major repositories covered.

Reply:
added the cli repo ✅